test/integration/freetext_editor_spec.js

```diff
@@ -2025,4 +2025,80 @@ describe("FreeText Editor", () => {
       );
     });
   });
+
+  describe("Focus must go on the current page", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait(
+        "tracemonkey.pdf",
+        ".annotationEditorLayer",
+        100,
+        async page => {
+          await page.waitForFunction("!!window.PDFViewerApplication.eventBus");
+          await page.evaluate(() => {
+            window.visitedPages = [];
+            window.PDFViewerApplication.eventBus._on(
```

Comment:
The `eventBus.{_on, _off}` format is only intended to be used *internally* within the viewer itself, so I think that we should use the regular format here since that ensures that all viewer-components get a change to handle the event first.
```suggestion
            window.PDFViewerApplication.eventBus.on(
```